src/inputs/selectNext/selectNext.tsx

```diff
@@ -19,6 +20,12 @@ import {
 import makeStyles from '../../styles/makeStyles';
 
 type PropTypes = {
+    /**
+    * Forces the Select component to always show the required indicator
+    * next to the label. The default behavior (if this prop is omitted or false) is for
+    * the required field indicator to disappear once a value has been selected.
+    */
+    alwaysShowRequiredIndicator: boolean;
```

Comment:
Should this be `alwaysShowRequiredIndicator?` (with trailing `?`) so that prop can be omitted and the default will be taken?

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/462/commits/a9b97e03bd4f25c81b8a7f8c2013bc6feb50f15c#diff-a86b373e2035b10f8f8a9550a5f1f02cc22e5be10743d74e35732aaaa88295b7L28-R28
Resolving.